packages/ui-core/src/components/color-adjuster/color-adjuster.component.test.tsx

```diff
@@ -1,33 +1,44 @@
 /* eslint-disable react/no-children-prop */
 import React from 'react';
-import { mount } from 'enzyme';
+import { render as rtlRender } from '@testing-library/react';
 
 import ColorAdjuster from './color-adjuster.component';
 
-describe('@keen.io/ui-core - <ColorAdjuster />', () => {
-  let mockFn: any;
+const render = (overProps: any = {}) => {
+  const props = {
```

Comment:
Could You please provide here a better typings like for eg. `Partial<ComponentProps<typeof ColorAdjuster>>`